Source/WebCore/animation/WebAnimation.h

```diff
@@ -237,6 +244,9 @@ class WebAnimation : public RefCounted<WebAnimation>, public EventTarget, public
     TimeToRunPendingTask m_timeToRunPendingPauseTask { TimeToRunPendingTask::NotScheduled };
     ReplaceState m_replaceState { ReplaceState::Active };
     uint64_t m_globalPosition { 0 };
+    std::variant<TimelineRangeOffset, RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>, String> m_rangeStart { };
+    std::variant<TimelineRangeOffset, RefPtr<CSSNumericValue>, RefPtr<CSSKeywordValue>, String> m_rangeEnd { };
```

Comment:
It's not clear to me what `{ }` initializes this value to by default.

Reply:
Changed this to initialize to normal.